.env.example

```diff
@@ -14,8 +14,8 @@ COVERALLS_REPO_TOKEN=covernal_token
 DATABASE_URL_PROD=production_url
 
 #
-AUTH_USER=Use_your_current_email
-AUTH_PASS=User_your_2-step-verification_Password_on_your_email
+AUTH_USER=amigosteam70@gmail.com
+AUTH_PASS=qauhadtdxpkwwdmk
```

Comment:
Please put those value in `.env` only and remain with their placeholders in `.env.example`

Reply:
All changes are added